app/models/stored_value.rb

```diff
@@ -0,0 +1,23 @@
+require 'base64'
+
+##
+# This class represents temporary storage for values processed in background.
+# This is a database solution, wich should not be the only one, as it has many limitations.
+
+class StoredValue < ApplicationRecord
+  ##
+  # Writes the value.
+  # It encodes the value in Base64, to be able to store any data.
+
+  def self.write(result_key, result_value, expire_at: nil)
+    record = self.find_by(key: result_key) || new(key: result_key)
+    record.value = Base64.encode64(result_value)
+    record.expire_at = expire_at || Time.now + 1.day
```

Comment:
Now I wonder if we don't limit us too much here, what if we want to use the storage for something that never expires? I like the fact it takes expire_at as argument, if we set `Time.now + 1.day` from place we actually write to store, it would be imho better. The expiration interval is probably always specific to what we store.

Reply:
Ok, but it allows permanent values! But done that :)